api/spec/requests/spree/api/images_controller_spec.rb

```diff
@@ -74,6 +74,16 @@ module Spree
           expect(response.status).to eq(204)
           expect { product_image.reload }.to raise_error(ActiveRecord::RecordNotFound)
         end
+
+        it "It returns nil attribute values and noimage urls when the image cannot be found",
```

Comment:
`It` is not required in the example text

Reply:
done!